### DISCUSSION TITLE
GrenadeTimer and AutoGrenadeThrow

### DISCUSSION THREAD

**Discussion by Samurai0111** [IceHacks/SurvivCheatInjector#807] [Ideas]
option to disable autogrenadethrow and add grenadetimer

**Comment by IceHacks** [comment]
I'm not aware of any AutoGrenadeThrow. Also, GrenadeTimer is already added. I'll make sure it is working soon.

**Comment by Samurai0111** [comment]
when you hold an grenade it will spam for you, add an option to disable that

**Comment by Samurai0111** [comment]
not related to the question but can you hide the open controls button

**Comment by IceHacks** [comment]
> not related to the question but can you hide the open controls button

Press `h`
